LICENSES/MIT.txt

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) <year> <copyright holders>
```

Comment:
The year and copyright holders need to be filled out.

Would you please check your other change lists for this mistake, too?

Thanks.

Reply:
That's not a mistake, according to https://reuse.software/faq/#license-templates; it's the 1st recommended option out of the two suggested there.